examples/cop-dem-glo-30/Copernicus_DSM_COG_10_N53_00_W115_00_DEM/Copernicus_DSM_COG_10_N53_00_W115_00_DEM.json

```diff
@@ -113,6 +124,129 @@
       "roles": [
         "data"
       ]
+    },
+    "SRC": {
```

Comment:
I can't recall if this is in the STAC spec or not, but I think it's more conventional to use lowercase asset names here.

Reply:
Seems to be convention particularly inherited from json. Was a little funky but found a solution.